Makefile

```diff
@@ -48,7 +48,11 @@ api:
 		--go-grpc_out=paths=source_relative:./api \
 		--validate_out=paths=source_relative,lang=go:./api \
 		--openapi_out=fq_schema_naming=true,default_response=false:. \
-		$(API_PROTO_FILES)"
+		$(API_PROTO_FILES) && \
+		cd api/ && \
```

Comment:
Does this mean that the make target changes the current working directory on the user's shell? If so, this is probably not what we want.